pallets/capacity/src/lib.rs

```diff
@@ -367,6 +378,26 @@ pub mod pallet {
 			});
 			Ok(())
 		}
+
+		/// Sets the epoch period length (in blocks).
+		///
+		/// # Requires
+		/// * Root Origin
+		///
+		/// ### Errors
+		/// - Returns `Error::MaxEpochLengthExceeded` if `length` is greater than T::MaxEpochLength.
+		#[pallet::call_index(3)]
+		#[pallet::weight(T::WeightInfo::unstake())]
+		pub fn set_epoch_length(
+			origin: OriginFor<T>,
+			length: T::BlockNumber,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+			ensure!(length <= T::MaxEpochLength::get(), Error::<T>::MaxEpochLengthExceeded);
```

Comment:
What do you think about emitting an event?
Something like, `EpochLengthUpdated`?

Reply:
Let's also add a ticket for adding an integration test to update the epoch length.